nixos/modules/services/continuous-integration/buildkite-agent.nix

```diff
@@ -101,7 +197,7 @@ in
 
     systemd.services.buildkite-agent =
       let copy = x: target: perms:
-                 "cp -f ${x} ${target}; ${pkgs.coreutils}/bin/chmod ${toString perms} ${target}; ";
+                 "${pkgs.coreutils}/bin/cp -f ${x} ${target}; ${pkgs.coreutils}/bin/chmod ${toString perms} ${target}; ";
```

Comment:
(nitpick) Maybe we could just add coreutils to `path = cfg.runtimePackages ++ [ pkgs.coreutils]` and we don't need to hardcode them here (for readability)?

Reply:
Good idea, thanks. I have fixed that.